pipeline/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -70,16 +70,20 @@
    },
    "source": [
     "## Minimal working example\n",
+    "Following files are required to run the mwe\n",
+    "1. The target vcf file to be QC\n",
+    "2. A reference vcf.gz file to generate the dbsnp_variants\n",
+    "3. A reference fasta file\n",
+    "\n",
     "\n",
     "```\n",
-    "sos run VCF_QC.ipynb dbsnp_annotate \\\n",
-    "    --genoFile data/00-All.vcf.gz\n",
+    "nohup sos run VCF_QC.ipynb dbsnp_annotate \\\n",
+    "    --genoFile data/00-All.renamechrs.vcf.gz &\n",
```

Comment:
dbsnp_annotate the first step should be using `00-All.vcf.gz` not `00-All.renamechrs.vcf.gz` because it is going to geenerate that file.